Nighttime-Operations/Weather-Constraints/index.rst

```diff
@@ -0,0 +1,179 @@
+
+The quantity that is more important than relative humidity is the **dew point temperature** and the **dew point difference**. 
+
+The dew point temperature is the ambient air temperature at which relative humidity will reach 100% - the air is completely saturated with water vapor.
+
+The dew point difference is the difference between the dew point and the coldest structure in the telescope dome. 
+
+Example:
+    Dew point temperature = 8 degrees C.
+
+    AuxTel M1 temperature = 6 degrees C.
+    
+    Glycol lines = 4 degrees C.
+    
+    Dew point difference = 4 degrees C.
+
+For any object or strucutre that is above the dew point temperature, there is less condensation risk.
+Water condenses on the coolest surface. 
+
+Dew point difference is > 2.5 degrees C:
+    - Normal operations may continue.
+
+Dew point difference is between 2 and 2.5 degrees C:
+    - Observers must consider closing. 
+        Go outside and check condensation on buildings and vehicles. 
+        If wet, advise to close.
+
+Dew point difference is 2 degrees C or less:
+    - Close immediately. 
+
+Observers must wait for the dew point temperature or humidity to recover from the closure limits for at least 15 minutes before opening again. 
+
+.. note::
+    If humidity has dropped during the night, before opening the dome slit, consider performing a quick dome drain procedure. 
+    Rotate the closed dome several times in both directions, exercisicing large slews. 
+    This can remove some of the water droplets that might have accumulated on the metal surface. 
+
+.. _weather-constraints-cloud-cover:
+
+Cloud cover
+^^^^^^^^^^^
+
+Until DIMM flux measurements are imported into Rubin TV or in the EFD, or the wavefront sensors of LSSTCam are available, determining safe or unsafe cloud coverage is a bit of an un-tested process.
+
+AuxTel has continued observing spectrae happily in 2.5 magnitudes of extinction, given that most targets are typically <= 8 magnitudes.
+
+The biggest threat of cloud cover is lower-atmosphere condensation, or virga, which is rain and moisture that falls from a cloud, but eavporates before it hits ground level.
+Virga appears like streaks of rain of whisps on the bottom of different types of cumulus clouds. 
+Winds can bring this moisture into the dome, or it can condense on the dome roof and fall through the slit. 
+
+If clouds are present in the all-sky camera - which is mounted next to AuxTel on Calibartion Hill - be mindful of further cloud accumulation and the direction they are coming from. 
+
+The `Gemini cloud cameras`_ are useful to evaluate how low the clouds are. 
+In the North East camera, verify that the clouds are not below the mountain peaks. 
+If clouds are at this low elevation, and they are overhead the telescope, these is a moisture risk.
+Cloud camera images update every 30 seconds during the night. 
```

Comment:
I think the flow might be improved by putting this and the next sentence after the first. Essentially put all the bits about the cameras first, then talk about what to look for when using them.

Great content by the way!

Reply:
👍